packages/pyright-internal/src/tests/testUtils.ts

```diff
@@ -152,15 +153,16 @@ export function bindSampleFile(fileName: string, configOptions = new ConfigOptio
 
 export function typeAnalyzeSampleFiles(
     fileNames: string[],
-    configOptions = new ConfigOptions('.')
+    configOptions = new ConfigOptions('.'),
+    console: ConsoleWithLogLevel | undefined = undefined
```

Comment:
This should be `console?: ConsoleWithLogLevel`. Easier to read and more concise.

Reply:
I believe that isn't allowed for parameters with default values. Yeah typescript raises an error:

```
Parameter cannot have question mark and initializer.
```

Reply:
The default value for a parameter with a `?` is implicitly `undefined`. So `console: ConsoleWithLogLevel | undefined = undefined` is just an unnecessarily verbose equivalent to `console?: ConsoleWithLogLevel`.

Reply:
Oh I thought that wasn't allowed. I thought if the previous parameter had a default, this one had to explicitly define it too.